test/test_circuit_tools.py

```diff
@@ -206,3 +206,27 @@ def test_circuit_to_data(self):
         actual_output = CircuitTools.circuit_to_data(circ, backend=self.dummy_backend)
 
         self.assertEqual(actual_output, target_output)
+
+    def test_convert_wire_order(self):
+        """test the convert_wire_order method"""
+
+        num_sites = 4
+        num_species = 3
+        wires_sequential = [0, 1, 4, 5, 8, 9]
+        wires_interleaved = [0, 3, 1, 4, 2, 5]
```

Comment:
This is a good example that this ordering is not a trivial issue. So essentially, position 1 in sequential becomes position 3 in interleaved because we have three species? Is it possible to write down the conversion rule as a one-lines comment?

Reply:
Added `# conversion rule: i -> (i % num_sites) * num_species + i // num_sites`